stdlib/universe/order_filter_group_flag_on_test.flux

```diff
@@ -0,0 +1,36 @@
+package universe_test
+
+import "testing"
+
+option now = () => (2030-01-01T00:00:00Z)
+
+input = "
+#datatype,string,long,dateTime:RFC3339,string,string,string,double
+#group,false,false,false,true,true,true,false
+#default,_result,,,,,,
+,result,table,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,system,host.local,load1,1.83
+,,0,2018-05-22T19:53:36Z,system,host.local,load1,1.63
+,,1,2018-05-22T19:53:26Z,system,host.local,load3,1.72
+,,2,2018-05-22T19:53:26Z,system,host.local,load4,1.77
+,,2,2018-05-22T19:53:36Z,system,host.local,load4,1.78
+,,2,2018-05-22T19:53:46Z,system,host.local,load4,1.77
+"
+
+output = "
+#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,double
+#group,false,false,false,false,true,false,false,false,false
+#default,_result,,,,,,,,
+,result,table,_start,_stop,_time,_measurement,host,_field,_value
+,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:26Z,system,host.local,load4,1.77
+,,1,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:36Z,system,host.local,load4,1.78
+,,2,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-22T19:53:46Z,system,host.local,load4,1.77
+"
+
+order_fn = (tables=<-) => tables
+    |> range(start: 2018-05-22T19:53:26Z)
+    |> group(columns: ["_time"])
+    |> filter(fn: (r) => r["_field"] == "load4")
+
+test order_evaluate = () =>
+    ({input: testing.loadStorage(csv: input), want: testing.loadMem(csv: output), fn: order_fn})
```

Comment:
These tests are great, could I trouble you to add a few more:
- tests similar to this but have `onEmpty: "keep"` in the call to `filter`
- A couple of tests with the following data set:

```
#datatype,string,long,string,string,dateTime:RFC3339,double
#group,false,false,true,true,false,false
#default,_result,0,Wx8gaJ,U2FstZ,,
,result,table,foo,bar,_time,_value

#datatype,string,long,string,string,dateTime:RFC3339,double
#group,false,false,true,true,false,false
#default,_result,1,U37BFq3,KYxg,,
,result,table,foo,bar,_time,_value

#datatype,string,long,string,string,dateTime:RFC3339,double
#group,false,false,true,true,false,false
#default,_result,2,U37BFq3,U2FstZ,,
,result,table,foo,bar,_time,_value

#datatype,string,long,string,string,dateTime:RFC3339,double
#group,false,false,true,true,false,false
#default,_result,3,Wx8gaJ,KYxg,,
,result,table,foo,bar,_time,_value
```
Those are four empty tables. It would be good to run the test with this input and make sure it behaves right when there are empty tables, for both filter with `onEmpty: "keep"` and `onEmpty: "drop"` (this latter one is the default)

Note that there would 8 `_test.flux` files in total, with both `on_test` and `off_test` variants of these four
- normal data set filter with `onEmpty: "keep"`
- normal data set filter with `onEmpty: "drop"`
- empty tables data set filter with `onEmpty: "keep"`
- empty tables data set filter with `onEmpty: "drop"`

I just want us to be certain that it's always safe to transpose group and filter to put the filter first, and I find it hard to reason about in my head.